pkg/restapi/edv/operation/operations.go

```diff
@@ -35,6 +35,8 @@ const (
 		docIDPathVariable + "}"
 )
 
+var logger = log.New("edv/pkg/restapi")
```

Comment:
Usually package names logging don't have / in the  name @fqutishat @bstasyszyn @sudeshrshetty should we have some recommendations about how to name logger for package

Reply:
Updated it to just be "restapi".